analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CreatingHashAlgorithms.cs

```diff
@@ -1,77 +1,104 @@
-﻿namespace NS
-{
-    using System.Security.Cryptography;
+﻿using System.Security.Cryptography;
 
-    public class TestClass
+namespace Tests.Diagnostics
+{
+    public class InsecureHashAlgorithm
     {
-        // RSPEC 4790: https://jira.sonarsource.com/browse/RSPEC-4790
-        public void ComputeHash()
+        public void Hash(byte[] temp)
         {
-            // Review all instantiations of classes that inherit from HashAlgorithm, for example:
-            HashAlgorithm hashAlgo = HashAlgorithm.Create();
-//                                   ^^^^^^^^^^^^^^^^^^^^^^    {{Make sure this weak hash algorithm is not used in a sensitive context here.}}
-            HashAlgorithm hashAlgo2 = HashAlgorithm.Create("SHA1");
```

Comment:
I can't find new version of this test. Can you add a line for each `unsafeAlgorithms` value?